api/s2n.h

```diff
@@ -69,6 +69,7 @@ extern int s2n_config_set_monotonic_clock(struct s2n_config *config, s2n_clock_t
 
 extern const char *s2n_strerror(int error, const char *lang);
 extern const char *s2n_strerror_debug(int error, const char *lang);
+extern const char *s2n_strerror_name(int error, const char *lang); 
```

Comment:
1 more nit: I don't expect the name change with lang, so do we need lang in here?

Reply:
Maybe not. I could remove the lang option.